app/controllers/publish/courses/fee_or_salary_controller.rb

```diff
@@ -28,6 +52,38 @@ def current_step
       def error_keys
         %i[funding_type program_type]
       end
+
+      def track_funding_type_changes
+        @funding_type_updated = params[:course][:funding_type] != @course.funding_type
```

Comment:
I tried to get it to work but i think we would need to customise the behaviour a little if we wanted to track this attribute alongside the others since the course model is already tracked with activerecord. I'll card up a refactor